### PR TITLE
replace deprecated _filter methods with _action methods

### PR DESCRIPTION
**Pull request by captainkevron**
Rails 5 deprecated the `_filter` methods and replaced them with the `_action` methods.

### CODE REVIEW

File: lib/localeapp/version.rb [hunk 1]
```diff
@@ -1,3 +1,3 @@
 module Localeapp
-  VERSION = "0.5.1"
+  VERSION = "0.6.0"
```
Comment (alanbrent) [inline lib/localeapp/version.rb]: Hmmm, there's already a [v0.6.0](https://rubygems.org/gems/localeapp/versions/0.6.0). Also, it looks to me like [our changes to this gem](https://github.com/jdxcode/localeapp/compare/master...Tapjoy:localeapp:master) may have been implemented slightly differently in [v0.6.10](https://github.com/Locale/localeapp/blob/master/CHANGELOG.md#version-0610). Perhaps we can look into updating TJS to use the upstream gem, the last in that series of releases (v0.6.14)? (Maybe even further!)

Comment (captainkevron) [reply lib/localeapp/version.rb]: Oh interesting, thanks for digging into that. I would be in favor of getting rid of our private fork if possible.

Comment (alanbrent) [reply lib/localeapp/version.rb]: Same. Let's do it :)

**Review by alanbrent — APPROVED** [review]
LGTM pending version change to `v0.5.1.rails5`

**Pull request merged**